lib/slack/responses/meetup.js

```diff
@@ -0,0 +1,39 @@
+// jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+const moment = require('moment');
+const Meetup = require('meetup-api');
+const meetup = Meetup({ key: process.env.MEETUP_API_TOKEN });
+
+const getLatestMeetup = (cb) => {
```

Comment:
Should this be in a separate file so it can also be called by the auto update task?

maybe lib/meetup/next-meeting